src/Main.hs

```diff
@@ -18,6 +19,11 @@ initialize wolfStates sheepStates = do
     return game
 
 
+-- assuming wolf always does the first step
+getFirstGameStep :: [Point] -> [Point] -> GameDTO
+getFirstGameStep wolfStates sheepStates = let gsFirst = initialize wolfStates sheepStates
+                                              in wolfMoveState (GameDTO (unsafePerformIO gsFirst, Unrecognized))
```

Comment:
Po co Ci unsafePerformIO?

Reply:
Żeby zrzutować IO GameDTO na zwykły typ GameDTO. To było najprostsze rozwiązanie, dlatego tego użyłam. I to jest też jedyne miejsce, gdzie używam unsafePerformIO.